build.gradle

```diff
@@ -199,13 +217,13 @@ private void setupLinuxArm32HfpTarget(Project project) {
 }
 
 private void setupIosTargets(Project project) {
-    project.apply plugin: IosPlugin
+    project.apply plugin: DarwinPlugin
 }
 
 import tasks.BuildIosSampleTask
 
-if (Target.currentTarget().isIos()) {
+if (Target.currentTarget().isDarwin()) {
     tasks.register('buildIosSample', BuildIosSampleTask) {
-        dependsOn ':sample-mpp-module:fatDebug', ':sample-mpp-module:fatRelease'
+        dependsOn ':sample-mpp-module:fatIosDebug', ':sample-mpp-module:fatIosRelease'
```

Comment:
It should be still named as "ios" right?

Reply:
Yes, sample has iOS version only now.